app/javascript/components/volume-mapping-form/volume-mapping-form.schema.js

```diff
@@ -112,6 +112,8 @@ const createSchema = (emsId, setEmsId, storageId, setStorageId, volumeId, setVol
       id: 'host_initiator_group_id',
       label: __('Host Initiator Group:'),
       isRequired: true,
+      helperText: __("Note! Volume can't be mapped to an empty host-initiator-group (host-cluster)." +
```

Comment:
```suggestion
      helperText: __("Note! Volume can't be mapped to an empty host-initiator-group (host-cluster).Empty host-initiator-groups are not listed."),
```

Reply:
@llivne Not sure why string concatenation is needed here. Any specific reason? Above suggestion is working for me.

Reply:
The IDE automaticly changed it because the line was long. I can set it like you suggested